ludwig/api.py

```diff
@@ -524,7 +525,7 @@ def on_epoch_end(self, trainer, progress_tracker, save_path):
                         self.config[TRAINER][BATCH_SIZE] = tuned_batch_size
                         trainer.batch_size = tuned_batch_size
 
-                    if self.config[TRAINER][EVAL_BATCH_SIZE] == AUTO:
+                    if self.config[TRAINER][EVAL_BATCH_SIZE] in {AUTO, None}:
```

Comment:
Does this mean we can get rid of [this extra condition](https://github.com/ludwig-ai/ludwig/blob/master/ludwig/trainers/trainer.py#L144)?

Reply:
Not quite, as the above code path is only executed when `batch_size=auto`